_includes/video.html

```diff
@@ -1,7 +1,13 @@
 <div class="video">
     {{ video.title }}
     <iframe width="560" height="315"
+        {% if video.youtube-id %}
         src="https://www.youtube-nocookie.com/embed/{{ video.youtube-id}}"
+        {% elsif video.vimeo-id %}
+        src="https://www.youtube-nocookie.com/embed/{{ video.youtube-id}}"
```

Comment:
shouldn't that be vimeo-id?

Reply:
Oh, good spot - copy/paster error. Will fix